.vscode/launch.json

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
This file should be removed before a merge.

Reply:
Heh, sure. I saw .idea too, so I thought why not.

Reply:
`.idea` is mostly because most of our developers use IntelliJ and it helps put the copyright on every new config. Feel free to submit a new PR for this.